src/components/AboutPage.js

```diff
@@ -0,0 +1,72 @@
+import * as React from "react";
+import NavigationBar from "./NavigationBar";
+
+export default function AboutPage () {
+    return (
+        <div className="flex flex-col h-screen bg-gray-100 ">
+            
+
+            <div className="flex-1 bg-maroon bg-opacity-80 flex flex-col justify-center items-center">
+                <h1 className='text-5xl text-pink-100 mt-16'>The Novelology Story</h1>
+                <h2 className='text-xl italic text-white mt-10 font-normal'>Turning pages, Uncovering Connections</h2>
+                <h2 className='text-xl italic text-white mt-10 font-normal'> </h2>
```

Comment:
Are these headers for later use?

Reply:
no, its just for this page

Reply:
Ok, are they there for spacing? theres two empty header blocks